proto/viam/app/v1/app.proto

```diff
@@ -669,6 +669,7 @@ message APIKey {
 
 message GetRobotAPIKeysResponse {
   repeated APIKey api_keys = 1;
+  repeated APIKeyWithAuthorizations api_keys_with_authorizations = 2;
```

Comment:
should I just fully break the API and do this?
```proto
message GetRobotAPIKeysResponse {
  repeated APIKeyWithAuthorizations api_keys = 1;
}
```

Reply:
i think fully break API

Reply:
updated